meshiphi/dataloaders/vector/abstract_vector.py

```diff
@@ -261,8 +261,13 @@ def calculate_coverage_from_df(bounds, data):
                 bounds_lat_range = bounds.get_lat_max() - bounds.get_lat_min()
                 # Get range of longitude values
                 data_long_range = data.long.max() - data.long.min()
-                bounds_long_range = bounds.get_long_max() - bounds.get_long_min()
-                # Calcualte area of each region
+                # If not going over antimeridian
+                if bounds.get_long_max() > bounds.get_long_min():
+                    bounds_long_range = bounds.get_long_max() - bounds.get_long_min()
+                # If goes over antimeridian
+                else:
+                    bounds_long_range = (180 - bounds.get_long_min()) + \
+                                        (bounds.get_long_max() + 180)                # Calcualte area of each region
```

Comment:
The comment here has been moved over to the side, just needs to go on a new line.